pylint/checkers/utils.py

```diff
@@ -455,6 +455,11 @@ def is_ancestor_name(
     return False
 
 
+def is_being_called(node: astroid.node_classes.NodeNG) -> bool:
```

Comment:
I like this style of doing things better than having super big conditional without explanation. Even though I know this won't be re-used it's nice to have a function if only for clarity.